module/applications/actor/api/multi-actor-sheet.mjs

```diff
@@ -0,0 +1,215 @@
+import ContextMenu5e from "../../context-menu.mjs";
+import BaseActorSheet from "./base-actor-sheet.mjs";
+
+/**
+ * An abstract class that implements functionality for sheets that contain multiple actors.
+ * @extends {BaseActorSheet}
+ */
+export default class MultiActorSheet extends BaseActorSheet {
```

Comment:
Limited sheet mode for both party and encounter could use a bit of love:

<img width="723" height="718" alt="Screenshot 2025-08-07 at 16 20 50" src="https://github.com/user-attachments/assets/15ca2efc-0c0f-4576-a782-1f246e7c0a58" />

<img width="516" height="166" alt="Screenshot 2025-08-07 at 16 16 32" src="https://github.com/user-attachments/assets/f05770b9-3a90-4500-9af0-9bf5bb735a56" />